File: engine/engine-model/src/main/scala/tools/packet/cashshop/operation/ShowBoughtCashPackageSuccess.scala
package tools.packet.cashshop.operation

import client.inventory.Item
import net.opcodes.SendOpcode
import tools.packet.PacketInput

class ShowBoughtCashPackageSuccess(private var _cashPacket: java.util.List[Item], private var _accountId: Int) extends PacketInput {
  def cashPacket: java.util.List[Item] = _cashPacket

  def accountId: Int = _accountId

  override def opcode(): SendOpcode = SendOpcode.CASH_SHOP_OPERATION
}

File: engine/engine-model/src/main/java/tools/packet/cashshop/operation/ShowBoughtCashPackageSuccess.java
package tools.packet.cashshop.operation;

import java.util.List;

import client.inventory.Item;
import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record ShowBoughtCashPackageSuccess(List<Item> cashPacket, Integer accountId) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.CASH_SHOP_OPERATION;
   }
}